src/main/java/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -60,6 +60,9 @@
   private boolean m_needToRecordBattleStatistics = true;
   private boolean m_needToCheckDefendingPlanesCanLand = true;
   private boolean m_needToCleanup = true;
+  private boolean needToFindRocketTargets = true;
+  private boolean needToFireRockets = true;
```

Comment:
It looks like if we put these variables in rocketHelper, much of the logic is simplified, and battleDelegate now longer has to keep track of that logic.  Placing the state in rocketHelper should also make it easier to verify in test code.